ts/screens/wallet/payment/NewTransactionSummaryScreen.tsx

```diff
@@ -478,6 +483,19 @@ const mapDispatchToProps = (dispatch: Dispatch, props: OwnProps) => {
       zendeskBlockedPaymentRptIdId,
       RptIdFromString.encode(rptId)
     );
+    // Add organization fiscal code custom field
+    addTicketCustomField(
+      zendeskPaymentOrgFiscalCode,
+      rptId.organizationFiscalCode
+    );
+    if (O.isSome(error)) {
+      // Add failure custom field
+      addTicketCustomField(zendeskPaymentFailure, error.value as string);
```

Comment:
Same as above.

Reply:
solved in [a7ea040](https://github.com/pagopa/io-app/pull/4431/commits/a7ea0408ac4eda03089d9c1d11fb8a9ad0e9b595)